### DISCUSSION TITLE
`--config-settings`?

### DISCUSSION THREAD

**Discussion by nschloe** [tox-dev/tox#3077] [Q&A] [answered]
When build my project with `pip`, I sometimes have to pass [config settings](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-C) like
```
pip install . -Cfoobar=123
```
(or equivalently `--config-settings=123`.) I need to do the same in my tox builds. Can I pass `--config-settings`?

_Edit:_ Created a feature request for it: https://github.com/tox-dev/tox/issues/3080

**Comment by jugmac00** [comment]
Have you tried using the `PIP_CONFIG_FILE` variable?

https://pip.pypa.io/en/stable/topics/configuration/#pip-config-file

**Reply by nschloe** [reply]
Not sure if `--config-settings` (which are settings for build backend) and `PIP_CONFIG_FILE` (settings for pip itself) serve the same purpose, i.e., if you can specify `--config-settings` in `PIP_CONFIG_FILE`s. Trying to find out now...

**Reply by nschloe** [reply]
I've made this into a feature request, https://github.com/tox-dev/tox/issues/3080.

**Reply by jugmac00** [reply, marked as answer]
Thank you, and sorry for the red herring. Solution will be tracked via #3080
